docs/conf.py

```diff
@@ -31,18 +31,14 @@
 import datetime
 from distutils.version import LooseVersion
 
-from sphinx import __version__
-SPHINX_LT_17 = LooseVersion(__version__) < LooseVersion('1.7')
-
 # -- Convert Sphinx Warnings to output to stdout not stderr---------------------
-if not SPHINX_LT_17:
-    import logging
-    from sphinx.util.logging import NAMESPACE, WarningStreamHandler, SafeEncodingWriter
+import logging
```

Comment:
Does this even work? Can we drop this hack of mine?

Reply:
Not a clue. I can remove it.

Reply:
Well the warnings are all there before so I'm gonna say no it doesn't.